tests/int/testCases/basic/apiFunction.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * @pact
+ * @pact-method GET
+ * @pact-path /api
+ */
+export function apiFunction() {
+    return 'string';
```

Comment:
Can we make this and other testCases more real by invoking HTTP request (via axios for example) in the function body and returning from it?

Reply:
I've created seperated testCase where I use axios interface.